mne_bids/tests/test_write.py

```diff
@@ -1228,6 +1230,11 @@ def test_eegieeg(dir_name, fname, reader, _bids_validate, tmp_path):
                           match='Encountered data in "int" format. '
                           'Converting to float32.'):
             bids_output_path = write_raw_bids(**kwargs)
+    elif dir_name == 'EGI':
+        with pytest.warns(RuntimeWarning,
+                          match=r'Encountered data in "(float|single)" '
+                          'format. Converting to float32.'):
```

Comment:
I'm not entirely sure if this is the correct way to use the match with a regex pattern, but it's required after https://github.com/mne-tools/mne-python/pull/10851 which fixes the `orig_format` for EGI datasets.

Reply:
I think that will work?  You can also include a `.*` after the disjunct if you want the pattern to fit on one line.

Reply:
I didn't like the idea to match everything of whatever length ;)